connector_equipment_service/models/agreement_serviceprofile.py

```diff
@@ -8,6 +8,8 @@
 class AgreementServiceProfile(models.Model):
     _inherit = ['agreement.serviceprofile']
 
+    equipment_id = fields.Many2one('maintenance.equipment', "Equipment ID")
```

Comment:
@osi-scampbell 

A dependency may be missing cause:
https://github.com/OCA/contract/blob/12.0/agreement_maintenance/models/agreement_serviceprofile.py#L10